### ISSUE TITLE
Please help. 

### ISSUE THREAD

**Issue by ph4t0o0o0om** [muaz-khan/RTCMultiConnection#649]
I have this error. "ICE failed, your TURN server appears to be broken, see about:webrtc for more details
Peer connection is closed between you &  1fw5ely6fboiksg83tj 
Object {  }
 state: failed RTCMultiConnection.min.js:18:29996
Peer connection is closed between you &  1fw5ely6fboiksg83tj 
Object {  }
 state: closed RTCMultiConnection.min.js:18:29996
Peer connection is closed between you &  1fw5ely6fboiksg83tj 
Object {  }
 state: closed" 

How to fix this?

**Comment by muaz-khan** [comment]
`webrtcweb.com` is down and `coTURN` and all other services (including `apache2`) are stopped. In the meanwhile, you can remove all links to coTURN:

Use following code in your HTML file:

```javascript
connection.iceServers = [{
        'urls': [
            'turn:webrtcweb.com:4455?transport=udp', // restund udp
        ],
        'username': 'muazkh',
        'credential': 'muazkh'
    },
    {
        'urls': [
            'stun:stun.l.google.com:19302',
            'stun:stun.l.google.com:19302?transport=udp',
        ]
    }
];
```

**If possible, please link & use third party TURN servers.**

----

I (accidentally) tried to upgrade ubuntu and it affected everything including `apt` tool. I asked my service-providers to format and install older version of ubuntu, again.

Because `apt` tool is not working, so I can not do anything myself.

**Comment by ph4t0o0o0om** [comment]
Sir it it possible to run a stun and turn server localy? And sir I have
email on you.

**Comment by muaz-khan** [comment]
I've fix coturn today. It should work now.

**Comment by ph4t0o0o0om** [comment]
Yes sir it is working now. But I want to try to run to my own local turn and stun server how to do that?  I already downloaded the simple-turn-windows-386.exe and try to run it but it only opens CMD in 1 second then suddenly close. What could I do for this?

**Comment by ph4t0o0o0om** [comment]
Sir do you really know how to use this as my stun and turn server? Link: http://www.stunprotocol.org/

**Comment by ph4t0o0o0om** [comment]
Okay sir. Now how can I used this? Do i need to change the codes on the IceServersHandler.js file? How?

**Comment by muaz-khan** [comment]
* https://www.rtcmulticonnection.org/docs/iceServers/

```javascript
// first step, ignore default STUN+TURN servers
connection.iceServers = [];

// second step, set STUN url
connection.iceServers.push({
    urls: 'stun:yourSTUN.com:port'
});

// last step, set TURN url (recommended)
connection.iceServers.push({
    urls: 'turn:yourTURN.com:port',
    credential: 'password',
    username: 'username'
});
```

**Comment by ph4t0o0o0om** [comment]
Okay sir I'm gonna check this out tommorow gonna have a rest for today. 
Anw sir do you know something that turn server on node.js? Because I already have on STUN (THe one you gave on the comment)

**Comment by rahul7827** [comment]
Hello Mr. Muaz, I too have same issue

On firefox console i found this error : 

I have this error. "ICE failed, your TURN server appears to be broken, see about:webrtc for more details
Peer connection is closed between you & 1fw5ely6fboiksg83tj

I tried the above solutions but none of them worked for me.

I am using RTCMultiConnection v3.4.4, also tried 3.6.9.

Issue is when i connect from the same network, streamer and viewer end works fine, while if streamer & viewer are on different network then viewer screen gets blank.

FYI : I am using "kurento media server" and nodejs.
